app/screens/settings/settings_separator.tsx

```diff
@@ -8,7 +8,7 @@ import {useTheme} from '@context/theme';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 
 const getStyleSheet = makeStyleSheetFromTheme((theme) => {
-    const groupSeparator = {
+    const groupSeparator: ViewStyle = {
```

Comment:
```suggestion
    const groupSeparator: StyleProp<ViewStyle> = {
```

Reply:
This is directly a style (we are passing the object) so there is no need to use the more generic StyleProp, I think.